tls/zcrypto_handshake_test.go

```diff
@@ -117,3 +119,106 @@ func TestCipherSuitesBadSSL(t *testing.T) {
 		})
 	}
 }
+
+func TestCipherSuitesLocalTLS13(t *testing.T) {
+	server := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`OK`))
+	}))
+	defer server.Close()
+
+	tests := []struct {
+		cipherSuite    uint16
+		expectedSuites []uint16
+		err            string
```

Comment:
Doesn't look like this is ever non-empty

Reply:
this is a placeholder. as we do expect there should be some errors when we start adding more ciphers from SSL3 etc